doc/devel/MEP/MEP14.rst

```diff
@@ -119,7 +121,7 @@ a core set of font selection parameters that will work across all text
 engines, and have engine-specific configuration to allow the user to
 do engine-specific things when required.  For example, it is possible
 to directly select a font by name in the "built-in" using
-`font.family`, but the same is not possible with "usetex".  It may be
+:rc:`font.family`, but the same is not possible with "usetex".  It may be
```

Comment:
Just as an idea: Would it make sense to have two variants of `:rc:` one with defaults and one without defaults?

e.g. to keep it short with/without defaults:
- `:rcd:`, `:rc:`
- or `:rc:`, `:rc_:`

Reply:
possibly (I guess :rc-: without the default would be somewhat mnemonic) but that's another project (and we can always grep for :rc: when doing that)